cms/static/js/views/pages/container.js

```diff
@@ -365,7 +366,13 @@ function($, _, Backbone, gettext, BasePage, ViewUtils, ContainerView, XBlockView
                     (useNewVideoEditor === "True" && blockType.includes("video"))||
                     (useNewProblemEditor === "True" && blockType.includes("problem"))
             ){
-                var destinationUrl = this.$('.xblock-header-primary').attr("authoring_MFE_base_url") + '/' + blockType[1] + '/' + encodeURI(data.locator);
+                var destinationUrl;
+                if (useVideoGalleryFlow === "True" && blockType.includes("video")) {
+                    destinationUrl = this.$('.xblock-header-primary').attr("authoring_MFE_base_url") + '/videos';
```

Comment:
@ChrisChV You will need to add the block location here too, since the video gallery will redirect to the video edit page and pre-fill the fields there.

```suggestion
                    destinationUrl = this.$('.xblock-header-primary').attr("authoring_MFE_base_url") + '/course-videos' + '?blockType=' + blockType[1] + '&location=' + encodeURI(data.locator);;
```

Reply:
@giovannicimolin Sure, thanks for the feedback!